### PR TITLE
Add a space

### PR DESCRIPTION
**Pull request by jessfraz**
### Why:

- There is a typo

Screen Shot 2021-03-08 at 11 24 48 AM

### What's being changed:

- Added a space

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by guggerjulien** [comment]
Je comprends pas

**Review by guggerjulien — COMMENTED** [review]
Je comprends pas.......

**Review by lee-dohm — APPROVED** [review]
Thanks for your help!

**Comment by guggerjulien** [comment]
good evening I don't understand how to get there

**Review by luismedicina23 — COMMENTED** [review]
@Maintenance Error sorry

**Comment by katanakrypto** [comment]
Is wohl zu langweilig da hiner euern bildschirmen

guggerjulien <notifications@github.com> schrieb am Mi., 10. März 2021,
02:41:

**Comment by katanakrypto** [comment]
Ok kommt zurück wie boomerang

luismedicina23 <notifications@github.com> schrieb am Mi., 10. März 2021,
03:47:

File: content/github/setting-up-and-managing-organizations-and-teams/synchronizing-a-team-with-an-identity-provider-group.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ versions:
 
 {% if currentVersion == "free-pro-team@latest" %}Team synchronization does not support IdP groups with more than 5000 members.{% endif %}
 
-Once a {% data variables.product.prodname_dotcom %} team is connected to an IdP group, your IdP administrator must make team membership changes through the identity provider. You cannot manage team membership on {% data variables.product.product_name %}{% if currentVersion == "free-pro-team@latest" %}or using the API{% endif %}.
```
Comment (luismedicina23) [inline content/github/setting-up-and-managing-organizations-and-teams/synchronizing-a-team-with-an-identity-provider-group.md:21]:
```suggestion
Once a {% data variables.product.prodname_dotcom %} team is connected to an IdP group, your IdP administrator must make team membership changes through the identity provider. You cannot manage team membership on {% data variables.product.product_name %}{% if currentVersion == "free-pro-team@latest" %}or using the API{% endif %}.
```

```

Comment (luismedicina23) [inline content/github/setting-up-and-managing-organizations-and-teams/synchronizing-a-team-with-an-identity-provider-group.md:21]:
```suggestion
Once a {% data variables.product.prodname_dotcom %} team is connected to an IdP group, your IdP administrator must make team membership changes through the identity provider. You cannot manage team membership on {% data variables.product.product_name %}{% if currentVersion == "free-pro-team@latest" %}or using the API{% endif %}.
```

**Review by luismedicina23 — COMMENTED** [review]
@Maintenance

**Comment by janiceilene** [comment]
The spam here is outrageous 😭 I'm going to lock this PR. Thanks again @jessfraz

**Pull request merged**